patches/react-native+0.64.2.patch

```diff
@@ -1,3 +1,24 @@
+diff --git a/node_modules/react-native/Libraries/Components/ScrollView/ScrollView.js b/node_modules/react-native/Libraries/Components/ScrollView/ScrollView.js
+index a069a24..a21fa38 100644
+--- a/node_modules/react-native/Libraries/Components/ScrollView/ScrollView.js
++++ b/node_modules/react-native/Libraries/Components/ScrollView/ScrollView.js
+@@ -1222,9 +1222,15 @@ class ScrollView extends React.Component<Props, State> {
+         // Note: we should split props.style on the inner and outer props
+         // however, the ScrollView still needs the baseStyle to be scrollable
+         const {outer, inner} = splitLayoutProps(flattenStyle(props.style));
++        let inverted;
++        if (inner.scaleY) {
++          inverted = {scaleY: -1};
++          delete inner['scaleY']
++        }
++
+         return React.cloneElement(
+           refreshControl,
+-          {style: [baseStyle, outer]},
++          {style: [baseStyle, outer, inverted]},
```

Comment:
This is the patch that allows us to bring back the pull to refresh indicator on android that respects the inverted control